survive-the-monsters.js

```diff
@@ -171,16 +232,75 @@ function despawnEnemy(enemyId) {
     }
 }
 
-const spawnRadius = Math.max(SCENE_WIDTH, SCENE_HEIGHT);
-const spawnRadiusHalf = spawnRadius / 2;
-setInterval(() => {
-    const x = playerX - spawnRadiusHalf + spawnRadius * Math.random();
-    const y = playerY - spawnRadiusHalf + spawnRadius * Math.random();
-    spawnEnemy(x, y, 0);
-}, 2000);
+function spawnBullet(x, y, type, initialVelocity) {
+    if (bulletsLargestId == BULLETS_COUNT_MAX) {
+        return;
+    }
+    let bulletId = bulletsLargestId + 1;
+    for (let id = 0; id < bulletId; id++) {
+        if (!bulletsState[id]) {
+            bulletId = id;
+            break;
+        }
+    }
+    bulletsLargestId = Math.max(bulletsLargestId, bulletId);
+    if (bulletsLargestId == BULLETS_COUNT_MAX) {
+        alert("out of ammo. sorry");
+    }
+    bulletsState[bulletId] = 1;
+    bulletsType[bulletId] = type;
```

Comment:
**suggestion**: Consider using a more user-friendly way to restart the game, such as a button or a menu, instead of an alert box.

Reply:
that will be next - mvp is done